hatice-ecevit-2016400138/assignment1/Anagram.java

```diff
@@ -0,0 +1,69 @@
+ import java.util.Arrays;
+ import java.util.HashMap;
+public class Anagram {
+
+    public static void main(String[] args) {
+        System.out.println(areAnagrams("apple","appll",true));
+        System.out.println(areAnagrams("listen","Silent",false));
+        System.out.println(areAnagramSentences("hi there hello","iH tHree lloeh",false));
+    }
+    public static boolean areAnagrams(String s1, String s2, boolean caseSensitive){
```

Comment:
Add javadoc comments for all your public methods (except `main()`).
If you expect someone else to call these methods, you want to document them.